docs/upgrade/17_to_18/extract_es_data.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+# Elasticsearch parameters
+ES_HOST="http://localhost:9200"
+INDEX="opencast_series"
+SIZE=1000
+OUTPUT_FILE="update_statements.sql"
+ERROR_LOG="error_log.txt"
+PAGE=0
+ES_USER="your_username"
+ES_PASS="your_password"
+
+# Clear output and error log files
+> "$OUTPUT_FILE"
+> "$ERROR_LOG"
+
+while :; do
+    FROM=$((PAGE * SIZE))
+    QUERY='{"_source": ["creator", "uid", "organization"], "from": '"$FROM"', "size": '"$SIZE"', "query": {"match_all": {}}}'
+
+    RESPONSE=$(curl -s -u "$ES_USER:$ES_PASS" -X POST "$ES_HOST/$INDEX/_search" -H "Content-Type: application/json" -d "$QUERY")
+
+    HITS_COUNT=$(echo "$RESPONSE" | jq '.hits.hits | length')
```

Comment:
Can we assume `jq` is installed on the host? Maybe mention the CLI tools required for this script in the release notes?